projects/services-ogc/src/lib/owc/owc-json.service.ts

```diff
@@ -259,8 +361,16 @@ export class OwcJsonService {
       const name = d.name;
       if (name === 'time') {
         dims.time = this.getTimeDimensions(resource.properties.dimensions);
+        /** if dimensions are defined but the values are null */
+        if (dims.time.values === null) {
```

Comment:
This is if someone don't want to configure all time dimensions, e.g. a lot of timesteps.
If the dimensions are set and values is null the client tries to find the GetCapabilities operation and fetch the dimensions for the layer afterwards. This is only working if the Server allows CORS!